src/index.css

```diff
@@ -11,3 +11,159 @@ code {
   font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
     monospace;
 }
+
+/*Page styling - background and albums*/
+body {
+  background-color: rgb(0, 0, 0);
+  box-sizing: border-box;
+  text-align: center;
+}
+
+/*Have trouble with the grid being to large - have to scroll horizontly - would fix if I had more time*/
+.albumWrapper {
+  margin-top: 50px;
+  display: grid;
+  grid-gap: 40px;
+  grid-template-columns: repeat(1, 1fr);
+  margin-bottom: 50px;
+}
+
+header {
+  color: whitesmoke;
+  margin-top: 28px;
+  font-size: 25px;
+  margin-bottom: 40px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 1em 0em;
+  font-weight: 700;
+}
+
+/*ALBUM NAME AND ARTIST NAME and HOVER*/
+
+.albumName {
+  font-family: Helvetica, Arial, sans-serif;
+  font-size: 14px;
+  color: #ffffff;
+  font-weight: bold;
+}
+
+.albumName:hover {
+  text-decoration: underline;
+}
+
+.artistName {
+  font-family: Helvetica, Arial, sans-serif;
+  font-size: 14px;
+  color: #a0a0a0;
+  font-weight: bold;
+}
+
+.artistName:hover {
+  text-decoration: underline;
+}
+
+a {
+  text-decoration: none;
+}
+
+
+/*ALBUM AND ICON BUTTONS*/
+
+.coverImage {
+  position: relative;
+  display: inline-block;
+  transition: 0.3s ease;
+}
+
+.ikonButtons {
+  /*placement*/
+  position: absolute;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100%;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  /*effects*/
+  opacity: 0;
+  pointer-events: auto;
+  transition: transform 0.3s ease, filter 0.3s ease;
+}
+
+.buttonPlay {
+  max-width: 35px;
+  margin: 0 15px;
+  filter: invert();
+}
+
+.buttonHeart,
+.buttonDots {
+  max-width: 25px;
+  margin: 0 15px;
+  filter: invert();
+}
+
+/* HOVER EFFECTS BUTTONS and ALBUM*/
+.ikonButtons:hover {
+  filter: brightness(1);
+  opacity: 1;
+  transform: scale(1.3);
+}
+
+.buttonPlay:hover {
+  transform: scale(1.5);
+}
+
+.buttonHeart:hover {
+  transform: scale(1.5);
+}
+
+.buttonDots:hover {
+  transform: scale(1.5);
```

Comment:
This is smart! I wonder if this is the way it should be done so the other icons don't react by moving as well (mine move as I solved this differently and couldn't figure out the issue). Another thing I need to try fixing in my own code!